types/transaction.go

```diff
@@ -1,12 +1,45 @@
 package types
 
 import (
+	"fmt"
 	"math/big"
 	"sync/atomic"
 
 	"github.com/0xPolygon/polygon-edge/helper/keccak"
 )
 
+// TransactionType represents the value between 0x00 and 0x7F to represents the type of transaction
+// TransactionType is put before transaction data in RLP encoded
+type TransactionType uint8
+
+const (
+	TxTypeLegacy TransactionType = 0x0
+	// TxTypeAccessList TransactionType = 0x1
+	// TxTypeDynamicFee TransactionType = 0x2
+	TxTypeState TransactionType = 0x7f
```

Comment:
Why are these two commented out?

Did we follow any EIP guidelines for this?

Reply:
These two commented types are introduced into Ethereum. I'm not sure we introduce them. But should avoid to use them

Reply:
@zivkovicmilos

Reply:
Agreed, I would leave them uncommented, and add a comment above them saying that they're unsupported and a link to their respective EIPs.

Implementing this is waaaay out of the scope of this PR 👍